src/hooks/useNotifications.ts

```diff
@@ -128,32 +128,48 @@ export const useNotifications = (colors: boolean): NotificationsState => {
                 data.map(async (accountNotifications) => {
                   return {
                     hostname: accountNotifications.hostname,
-                    notifications: await axios.all<Notification>(
-                      accountNotifications.notifications.map(
-                        async (notification: Notification) => {
-                          const isEnterprise = isEnterpriseHost(
-                            accountNotifications.hostname,
-                          );
-                          const token = isEnterprise
-                            ? getEnterpriseAccountToken(
-                                accountNotifications.hostname,
-                                accounts.enterpriseAccounts,
-                              )
-                            : accounts.token;
-
-                          const additionalSubjectDetails =
-                            await getGitifySubjectDetails(notification, token);
-
-                          return {
-                            ...notification,
-                            subject: {
-                              ...notification.subject,
-                              ...additionalSubjectDetails,
-                            },
-                          };
-                        },
-                      ),
-                    ),
+                    notifications: await axios
+                      .all<Notification>(
+                        accountNotifications.notifications.map(
+                          async (notification: Notification) => {
+                            const isEnterprise = isEnterpriseHost(
+                              accountNotifications.hostname,
+                            );
+                            const token = isEnterprise
```

Comment:
IMHO this indentation level has crossed a line where something should be split into smaller pieces.

Reply:
Agree - I too would really like to refactor most of this file into smaller pieces.